imgur-screenshot.sh

```diff
@@ -103,6 +117,49 @@ if [ "${1}" = "--check" ]; then
   exit 0
 fi
 
+# Install ubuntu application launcher icon for this user
+if [ "${1}" = "--install-ubuntu" ]; then
+  if [ -d ~/.local/share/applications/ ]; then
+    DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+    curl --output "$DIR/imgur.ico" "https://imgur.com/favicon.ico" &> /dev/null
+    cat <<EOF | awk '{$1=$1};1'> ~/.local/share/applications/imgur-screenshot.desktop
+      #!/usr/bin/env xdg-open
+      [Desktop Entry]
+      Name=Imgur Screenshot
+      Comment=Launch Imgur Screenshot
+      Exec=$DIR/imgur-screenshot.sh -o false
```

Comment:
Is there a reason you're overriding the `open` configuration in these `Exec` commands?

Reply:
This is a personal preference that I didn't really know I should expose. In my workflow, I only use imgur-screenshot to snag interesting bits of my screen for communicating with others. I never need to open the file for viewing because all I want is a URI of what I'm currently looking at. I added the Edit and Fullscreen options because those are use cases I could see others using.  The delete is in there because I periodically want to delete all of the artifacts from imgur because the communication is done and it just doesn't need to be there anymore.

I guess I just don't know what workflows I should support. Would it make sense to add the open flag for the default action and leave the other menu items as is?

Reply:
The edit and fullscreen menu items are fine to be included, but IMO none of them should use the open flag. If users don't want images to be opened, they can change that in their config.